service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/RequestStatementMapper.java

```diff
@@ -125,7 +125,7 @@ private void processAgent(@NonNull Reference agent, Reference onBehalfOf) {
                 final String participant = participantMapper.mapToParticipant(participantRef, ParticipantType.AUTHOR);
                 templateParameters.participant(participant);
 
-            } else if (isReferenceToType(agent, ResourceType.Practitioner) || isReferenceToType(agent, ResourceType.Organization)) {
+            } else if (isReferenceToType(agent, ResourceType.Practitioner)) {
```

Comment:
I'd not used this for parity with the other disjunct but now we can be more direct
```suggestion
            } else if (isReferenceToPractitioner(agent)) {
```